change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Moved code",
```

Comment:
Can we be specific in this comment about which code was removed?
Helps when we go over the combined changelog later.

Reply:
This is a no-op for this package (nothing changes from the perspective of the composites package) so type is 'none'. When the changelog is generated it omits these

Reply:
Oh cool. Didn't know that.